modules/ding_wayf_dk/modules/ding_wayf_dk_create_user/ding_wayf_dk_create_user.info

```diff
@@ -0,0 +1,6 @@
+name = Ding WAYF (DK) create uers
+description = Implementation of user creation using (wayf.dk).
+core = 7.x
+package = Ding!
+dependencies[] = ding_wayf_dk
+dependencies[] = ding_wayf_dk_login
```

Comment:
`ding_wayf_dk_login` already depends on `ding_wayf_dk`. Is there a reason to return it twice?